src/components/modal.js

```diff
@@ -0,0 +1,207 @@
+/** @jsx jsx */
+import React, { Component } from 'react';
+import { jsx, Flex, Styled } from 'theme-ui';
+import ReactModal from 'react-modal';
+import { X, ArrowLeft } from 'react-feather';
+
+import IconButton from './icon-botton';
+import Text from './text';
+
+ReactModal.setAppElement('body');
+
+class ModalWrapper extends Component {
```

Comment:
I'd prefer it if we implement this as a functional component with hooks, but we can do that at a later stage.

Reply:
I will have a look at this, and update it to hooks.

Reply:
Updated to hooks